src/components/Info.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Article = styled.article`
+  width: 100%;
+  max-width: 739px;
+  margin: 130px auto 0;
+  padding: 0 20px 185px;
+
+  > p {
+    line-height: 26.98px;
+    letter-spacing: 0.03px;
+  }
+`;
+
+const Heading = styled.h2`
+  font-size: 24px;
+  font-family: ${({ theme }) => theme.font.secondary};
+  color: ${({ theme }) => theme.color.secondary};
+  margin-bottom: 14px;
+`;
+
+const List = styled.ul`
+  margin: 0px 0 105px;
+  padding: 0;
+  list-style-position: inside;
+`;
+
+const ListItem = styled.li`
```

Comment:
The styles for the `li` are targeting the dot, the styles for the span the text, right? A suggestion to make this a bit more elegant imo would be to use `:before` with `list-style: none` on the `ul`. The `li` styles could look similar to this:

```
styled.li`
  padding-left: ??;
  ...

  :before {
    content: "•";
    position: absolute;
    left: ??;
  }
`
``` 

What do you think?

Reply:
Yes!  I was trying to come up with an elegant way and never quite got there.  I like this.